carrier.go

```diff
@@ -24,19 +24,20 @@ type CarrierFields struct {
 // CarrierAccount encapsulates credentials and other information related to a
 // carrier account.
 type CarrierAccount struct {
-	ID              string            `json:"id,omitempty"`
-	Object          string            `json:"object,omitempty"`
-	Reference       string            `json:"reference,omitempty"`
-	CreatedAt       *time.Time        `json:"created_at,omitempty"`
-	UpdatedAt       *time.Time        `json:"updated_at,omitempty"`
-	Type            string            `json:"type,omitempty"`
-	Fields          *CarrierFields    `json:"fields,omitempty"`
-	Clone           bool              `json:"clone,omitempty"`
-	Description     string            `json:"description,omitempty"`
-	Readable        string            `json:"readable,omitempty"`
-	Credentials     map[string]string `json:"credentials"`
-	TestCredentials map[string]string `json:"test_credentials"`
-	BillingType     string            `json:"billing_type,omitempty"`
+	ID               string                 `json:"id,omitempty"`
+	Object           string                 `json:"object,omitempty"`
+	Reference        string                 `json:"reference,omitempty"`
+	CreatedAt        *time.Time             `json:"created_at,omitempty"`
+	UpdatedAt        *time.Time             `json:"updated_at,omitempty"`
+	Type             string                 `json:"type,omitempty"`
+	Fields           *CarrierFields         `json:"fields,omitempty"`
+	Clone            bool                   `json:"clone,omitempty"`
+	Description      string                 `json:"description,omitempty"`
+	Readable         string                 `json:"readable,omitempty"`
+	Credentials      map[string]string      `json:"credentials"`
+	TestCredentials  map[string]string      `json:"test_credentials"`
+	RegistrationData map[string]interface{} `json:"registration_data"`
```

Comment:
These all need `,omitempty` added since it could be either or and some may have prod or test creds and not the other.

Reply:
Does sending a "null" to the API versus not sending it at all make a difference?

Reply:
It could. In this lib, just about every field has `omitempty` so we should follow convention here.